GrandCentralBoard/Widgets/Slack/SlackMessage.swift

```diff
@@ -10,4 +10,9 @@ import Foundation
 
 struct SlackMessage {
     let text: String
+    let timestamp: NSDate
+
+    var channel: String?
+    var author: String?
```

Comment:
I'm not sure that `SlackMessage` can exist without author. Who wrote the message?